libevm/VM.cpp

```diff
@@ -44,6 +44,40 @@ template <class S> S modWorkaround(S const& _a, S const& _b)
 
 
 //
+// for decoding destinations of JUMPTO, JUMPV, JUMPSUB and JUMPSUBV
+//
+
+uint64_t VM::decode_jump_dest(const byte* const _code, uint64_t& _pc)
+{
+	// turn 4 MSB-first bytes in the code into a native-order integer
```

Comment:
Can you move general comments about the function above the function definition (Doxygen style)?